packages/trader/src/AppV2/Containers/ContractDetails/__tests__/contract-details.spec.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { act, render, screen, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import ContractDetails from '../contract-details';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import useOrderDetails from 'AppV2/Hooks/useOrderDetails';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../trader-providers';
+import moment from 'moment';
+import {
+    isValidToSell,
+    isMultiplierContract,
+    isValidToCancel,
+    isOpen,
+    useWS,
+    WS,
+    hasContractEntered,
+    isForwardStarting,
+    isAccumulatorContract,
+} from '@deriv/shared';
+import { getContractDetailsConfig } from 'AppV2/Utils/contract-details-config';
+
+jest.mock('AppV2/Hooks/useContractDetails', () => jest.fn());
+jest.mock('AppV2/Hooks/useOrderDetails', () => jest.fn());
+
+jest.mock('AppV2/Components/ContractCard', () => {
+    const ContractCard = () => <div data-testid='contract-card'>ContractCard</div>;
```

Comment:
We can find it by text, so why in some cases we are adding `data-testid`?

Reply:
removed thanks